src/test/java/core/basesyntax/service/TransactionProcessDataServiceImplTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.transaction.BalanceTransactionHandler;
+import core.basesyntax.service.transaction.PurchaseTransactionHandler;
+import core.basesyntax.service.transaction.ReturnTransactionHandler;
+import core.basesyntax.service.transaction.SupplyTransactionHandler;
+import core.basesyntax.service.transaction.TransactionHandler;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TransactionProcessDataServiceImplTest {
+    private static final String FILE_PATH = "src/main/java/core/basesyntax/db/source.csv";
+    private static final String INCORRECT_OPERATION_FILE_PATH =
+            "src/main/java/core/basesyntax/db/sourceIncorrectOperation.csv";
+    private static final String INCORRECT_QUANTITY_FILE_PATH =
+            "src/main/java/core/basesyntax/db/sourceIncorrectQuantity.csv";
+    private static final String HEAD_VALUE = "type,fruit,quantity";
+    private static TransactionProcessDataService transactionProcessDataService;
+    private static Reader reader = new ReaderImpl();
```

Comment:
initialize this service in a method annotated with Before or BeforeClass (depends on the version of junit)